### ISSUE TITLE
[node v6.2.2] [npm  v3.9.5] sh: install-engine-dependencies: command not found

### ISSUE THREAD

**Issue by leoj3n** [stealjs/steal-tools#484]
Project source: https://github.com/stealjs/steal/wiki/Getting-Started

`npm install`:
~~~
npm ERR! node v6.2.2
npm ERR! npm  v3.9.5
~~~

`package.json`: (notice `"steal": "stealjs/steal#major"`)
~~~
{
  "name": "myhub",
  "version": "1.0.0",
  "description": "",
  "main": "myhub.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "bootstrap": "^3.3.6",
    "jquery": "^3.0.0",
    "justifiedGallery": "^3.6.2",
    "steal": "stealjs/steal#major"
  },
  "devDependencies": {
    "steal-qunit": "^0.1.1",
    "steal-tools": "stealjs/steal-tools#major"
  },
  "system": {
     "map": {
         "justifiedGallery": "justifiedGallery/src/js/justifiedGallery"
     },
     "meta": {
         "justifiedGallery/src/js/justifiedGallery": {
             "format": "global",
             "deps": ["jquery","justifiedGallery/src/less/justifiedGallery.less"]
         }
     }
  }
}
~~~

`npm install`: (notice `sh: install-engine-dependencies: command not found`)
~~~
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

> fsevents@1.0.12 install /Users/leoj/workspace/stealjs/myhub/node_modules/fsevents
> node-pre-gyp install --fallback-to-build

gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node
  COPY /Users/leoj/workspace/stealjs/myhub/node_modules/fsevents/lib/binding/Release/node-v48-darwin-x64/fse.node
  TOUCH Release/obj.target/action_after_build.stamp

> steal-tools@1.0.0-beta.0 postinstall /Users/leoj/workspace/stealjs/myhub/node_modules/steal-tools
> install-engine-dependencies

sh: install-engine-dependencies: command not found
myhub@1.0.0 /Users/leoj/workspace/stealjs/myhub
└── steal@1.0.0-beta.0  (git://github.com/stealjs/steal.git#249a1272ae78d385c2eed7dcc6f343047559c4fc)

npm WARN myhub@1.0.0 No description
npm WARN myhub@1.0.0 No repository field.
npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/leoj/.nvm/versions/node/v6.2.2/bin/node" "/Users/leoj/.nvm/versions/node/v6.2.2/bin/npm" "update"
npm ERR! node v6.2.2
npm ERR! npm  v3.9.5
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! steal-tools@1.0.0-beta.0 postinstall: `install-engine-dependencies`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the steal-tools@1.0.0-beta.0 postinstall script 'install-engine-dependencies'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the steal-tools package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     install-engine-dependencies
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs steal-tools
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls steal-tools
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/leoj/workspace/stealjs/myhub/npm-debug.log
~~~

`npm-debug.log`: http://www.filedropper.com/npm-debug_1

Same error when using  `node v5.9.1` and `npm v3.8.8`, and/or `"steal-tools": "1.0.0-beta.0"`.

**Comment by leoj3n** [comment]
https://github.com/bitovi/engine-dependencies

https://github.com/stealjs/steal-tools/blob/c6f735639ddffe17fed8d35ea591e217c8ab87fa/package.json#L116

**Comment by leoj3n** [comment]
Can get npm to not fail installing `steal-tools` by manually installing `engine-dependencies` first, like:

~~~
npm install engine-dependencies --save-dev
~~~

👍

**Comment by leoj3n** [comment]
Should be fixed by #485; needed to use `prepublish`.
